zerver/management/commands/restore_messages.py

```diff
@@ -22,17 +22,23 @@ class Command(ZulipBaseCommand):
 
 Examples:
 To restore all recently deleted messages:
-  ./manage.py restore_messages
+  ./manage.py restore_messages --restore-all --restore
 To restore a specific ArchiveTransaction:
   ./manage.py restore_messages --transaction-id=1
 """
 
     def add_arguments(self, parser: CommandParser) -> None:
+        parser.add_argument(
+            "-A",
+            "--restore-all",
```

Comment:
I might just call this `--all`

Reply:
Yeah, me too, and no need for a short `-A` parameter either.